fpmsyncd/routesync.cpp

```diff
@@ -114,10 +118,17 @@ void RouteSync::onMsg(int nlmsg_type, struct nl_object *obj)
         }
     }
 
+    if (nexthops.size() == 0)
```

Comment:
can you check if they are interface subnet? If yes, we can ignore it, without print any info.